prow/github/client.go

```diff
@@ -408,6 +408,28 @@ func (c *Client) RemoveLabel(org, repo string, number int, label string) error {
 	return err
 }
 
+func (c *Client) AssignIssue(org, repo string, number int, logins []string) error {
+	c.log("AssignIssue", org, repo, number, logins)
+	_, err := c.request(&request{
+		method:      http.MethodPost,
+		path:        fmt.Sprintf("%s/repos/%s/%s/issues/%d/assignees", c.base, org, repo, number),
+		requestBody: map[string][]string{"assignees": logins},
```

Comment:
have you verified that this only appends to the existing assignees?  I think it may overwrite the list on github (I think we ran into this problem before)